src/review/views.py

```diff
@@ -275,18 +276,33 @@ def reviewer_decision(
             message=message,
             short_name='Assignment accepted',
         )
-        logic.notify_editors(submission, message, editors, user, 'review')
+        logic.notify_editors(
+            book=submission,
+            message=message,
+            book_editors=book_editors,
+            series_editor=series_editor,
+            creator=user,
+            workflow='review'
+        )
 
     elif decision and decision == 'decline':
         review_assignment.declined = timezone.now()
         message = (
-            "Review Assignment request for '%s' has been declined by %s %s." % (
-                submission.title,
-                review_assignment.user.first_name,
-                review_assignment.user.last_name,
+            "Review Assignment request for '{book_title}' has been declined by"
+            " {reviewer_first_name} {reviewer_last_name}.".format(
+                book_title=submission.title,
+                reviewer_first_name=review_assignment.user.first_name,
+                reviewer_last_name=review_assignment.user.last_name,
             )
         )
-        logic.notify_editors(submission, message, editors, user, 'review')
+        logic.notify_editors(
```

Comment:
If `logic.notify_editors` is always the same function, taking always the same arguments, we could find a way to make this a bit more DRY, probably with a dictionary expansion.